src/angular/controllers/ctrl-home-page-spotlight.js

```diff
@@ -0,0 +1,5 @@
+const app = angular.module('materia')
```

Comment:
Since the homepage was rewritten in React, this file isn't being used any more. I'll add another commit to remove it.